public/js/p3/store/IDMappingAppMemoryStore.js

```diff
@@ -141,9 +141,9 @@ define([
         else if (via =="refseq_locus_tag"){
           joinId = {"genome_feature":"refseq_locus_tag","id_ref":"Gene_OrderedLocusName"};
         }
-        else if (via =="protein_id"){
-          joinId = {"genome_feature":"protein_id","id_ref":"RefSeq"};
-        }
+        // else if (via =="protein_id"){
+        //   joinId = {"genome_feature":"protein_id","id_ref":"RefSeq"};
+        // }
```

Comment:
This block, i believe, is handling mapping strategy (advanced option in the form). So, we should keep it, i guess.